src/oxli/oxli_exception_convert.cc

```diff
@@ -0,0 +1,22 @@
+#include "Python.h"
+#include <exception>
+#include <string>
+#include "oxli/oxli_exception.hh"
+#include "oxli/oxli_exception_convert.hh"
+
+
+void oxli_raise_py_error()
```

Comment:
Implementation of the conversion from C++ to Python exceptions. These are the one I used in HLLCounter, but we will need to map all the others too.

Possible problem: methods that throw the same C++ exception, but it is mapped to different Python exceptions. Didn't see any examples yet, but if it happens we might need to have more than one of this mapping functions around.